### DISCUSSION TITLE
Github Actions for org (private) repo ?

### DISCUSSION THREAD

**Discussion by nojb** [community/community#25898] [Actions]
<p>Hello,</p>
<p>I was recently let in into the Github Actions beta, but I don’t see the “Actions” in the (private) repo of my organization. I am an Admin of the repo but not the owner of the organization.</p>
<p>Is it possible for me to use Github Actions in this situation?</p>
<p>Thanks!</p>

**Comment by DeepInThought** [comment, marked as answer]
<p>I had to sign-up my organization as well.</p>
